src/main/resources/messages/TestGenie.properties

```diff
@@ -1,10 +1,10 @@
 name=My Plugin
 applicationService=Application service
 projectService=Project service: {0}
-evosuiteTestGenerationMessage=EvoSuite: Generating tests...
-evosuiteBuildMessage=Building project...
-evosuiteSearchMessage=Searching for test suite...
-evosuitePostProcessMessage=Post-processing...
+evosuiteTestGenerationMessage=Abracadabra! Your wish has been used. Generating tests...
+evosuiteBuildMessage=Abracadabra! Your wish has been used. Building project...
+evosuiteSearchMessage=Abracadabra! Your wish has been used. The best tests are being generated...
+evosuitePostProcessMessage=You have no more wishes left. But now you have your tests
```

Comment:
"please enter your credit card information".
Maybe we wanna be a bit clearer here 🤣

Reply:
Don't forget to support me on Patreon...